services/121-service/test/registrations/duplicates/create-registration-distinctness-errors.test.ts

```diff
@@ -0,0 +1,80 @@
+import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
+import {
+  createRegistrationDistinctness,
+  getRegistrationIdByReferenceId,
+  importRegistrations,
+} from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import {
+  registrationPV5,
+  registrationPV6,
+} from '@121-service/test/registrations/pagination/pagination-data';
+const registration1 = { ...registrationPV5 };
+const registration2 = { ...registrationPV6 };
+const programId = 2;
+
+// This is seperate from the other test file because it is testing the error case
+// it also saves a bit of processing time because it does not have to reset db before each test
+
+describe('Unsuccessfully mark registrations a distinct from each other', () => {
+  let registrationId1: number;
+  let registrationId2: number;
+  let accessToken: string;
+
+  beforeAll(async () => {
+    await resetDB(SeedScript.nlrcMultiple);
+    accessToken = await getAccessToken();
+    registration1.phoneNumber = '1234567890';
+    registration2.phoneNumber = '1234567890';
```

Comment:
```suggestion
    const duplicatePhoneNumber = '1234567890';
    registration1.phoneNumber = duplicatePhoneNumber;
    registration2.phoneNumber = duplicatePhoneNumber;
```

Otherwise you could still change one-but-not-the-other... ;)

Reply:
resolved with rebase